src/ua/griddynamics/tictactoe/Main.java

```diff
@@ -4,14 +4,31 @@
 
 public class Main {
     public static void main(String[] args) {
-
         Scanner scanner = new Scanner(System.in);
-        System.out.println("Enter cells: ");
-        String input = scanner.nextLine();
-
-        Game game = new Game(input);
+        Game game = new Game("_________");
         game.showGrid();
 
+        while (!game.isWinner('X') && !game.isWinner('O') && game.isNotFinished()) {
```

Comment:
instead of calling separate methods isNotFinished can check if somebody wined or there is a draw